File: docs/snippets/elixir/getting-started/basic_usage.exs
```elixir title="Elixir"
# Basic document extraction workflow
# Load file -> extract -> access results

{:ok, result} = Xberg.extract("document.pdf")

IO.puts("Extracted Content:")
IO.puts(result.content)

IO.puts("\nMetadata:")
IO.puts("Format: #{inspect(result.metadata.format)}")
IO.puts("Tables found: #{length(result.tables)}")
```

File: docs/snippets/csharp/getting-started/basic_usage.cs
using Xberg;

var config = new ExtractionConfig
{
    UseCache = true,
    EnableQualityProcessing = true
};

var result = XbergClient.ExtractSync("document.pdf", config);

Console.WriteLine(result.Content);
Console.WriteLine($"MIME Type: {result.MimeType}");
